core/common/src/test/java/alluxio/wire/WorkerInfoTest.java

```diff
@@ -37,6 +40,23 @@ public void thrift() {
     checkEquality(workerInfo, other);
   }
 
+  @Test
+  public void checkLastContactSecComparator() {
+    int[] lastContactSecList = {55, 10, 2, 40, 100, 6};
```

Comment:
Given that this is a unit test for the comparator, not for sorting functionality, I think it would make more sense to directly call `LastContactSecComparator.compare(x, y)` for a few example values of `x` and `y`.